servers/src/main/java/tachyon/worker/block/BlockLockManager.java

```diff
@@ -120,9 +118,9 @@ public void unlockBlock(long lockId) throws NotFoundException {
     Lock lock;
     synchronized (mSharedMapsLock) {
       LockRecord record = mLockIdToRecordMap.get(lockId);
-      if (null == record) {
-        throw new NotFoundException("Failed to unlockBlock: lockId " + lockId
-            + " has no lock record");
+      if (record == null) {
```

Comment:
Although there is no concrete style in Tachyon, I prefer using `constant == variable` to prevent accidental assignment.